### PR TITLE
Astro 3/7 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/team-infra-61113e60aad91d000feb81bf/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODkwNDI

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))
 
+## March 7, 2023
+
+### Book office hours from the Cloud UI
```
Comment (jwitz) [inline astro/release-notes.md:28]:
```suggestion
### Get expert advice on Astro and Airflow in office hours
```

File: astro/astro-support.md [hunk 2]
```diff
@@ -116,3 +116,9 @@ If you've submitted your support request on the Astronomer support portal, sign
 To add a teammate to an existing support request, cc them when replying on the support ticket email thread.
 
 :::
+
+## Book office hours in the Cloud UI
+
+An alternative to submitting a support ticket is meeting with the Astronomer data engineering team during office hours. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines.
```
Comment (kvandoorn) [inline astro/astro-support.md:122]: Outside of break-fix support, you can meet with an Astro Data Engineer team member during scheduled office hours. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines.

File: astro/release-notes.md [hunk 3]
```diff
@@ -23,6 +23,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+Office hours are a new way for Astro customers to meet with the Astronomer data engineering team. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines.
```
Comment (kvandoorn) [inline astro/release-notes.md:30]: Office hours are a new way for Astro customers to meet with the Astro Data Engineering team. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines. Please complete the prompts and provide appropriate information about prior support tickets, communication, and any issues.

**Review by kvandoorn — CHANGES_REQUESTED** [review]

**Review by lzdanski — APPROVED** [review]
I like the wording update for the office hours description - the position at the end in the support page had implied a sort of sequence of steps, but this clears that up.

File: astro/resource-reference-gcp.md [hunk 4]
```diff
@@ -87,6 +87,7 @@ The following table lists all available instance types for worker node pools, as
 
 | Node Instance Type | CPU     | Memory      |
 | ------------------ | ------- | ----------- |
+| e2-standard-2      | 2 CPUs  | 8 GiB MEM  |
```
Comment (chomipi88) [inline astro/resource-reference-gcp.md:90]: we did not release support for this yet.

**Pull request merged**